docs/source/contents/4_draft_mapping.rst

```diff
@@ -1331,6 +1331,297 @@ location
+future, presently it seems unlikely that it will. More importantly, if such a project became a priority, it would not be
+difficult to distinguish via code UTK's existing English records from records in another language. If we did want to create
+a project like this, information on the language of cataloging could be added across the repository with minimal effort.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:languageOfCataloging"
+
+Decision
+^^^^^^^^
+
+Since we are not currently utilizing these values in any way, these should be dropped in the mapping.
+
+Here's an `example record - sanborn:1002 <https://digital.lib.utk.edu/collections/islandora/object/sanborn%3A1002/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <recordInfo>
+        <languageOfCataloging>
+            <languageTerm authority="iso639-2b" type="code">eng</languageTerm>
+        </languageOfCataloging>
+    </recordInfo>
+
+recordOrigin
+------------
+
+Use Case
+^^^^^^^^
+
+The <recordOrigin> element includes information about what methods or transformations were used to prepare a record. There
+are six different distinct values in UTK's metadata.
+
+Justification
+^^^^^^^^^^^^^
+
+Because the existing values all relate to MODS XML, the string values present will no longer be applicable in a RDF-based
+platform. Discussion indicated that there might be some use for the general property of recordOrigin if a link to this
+mapping document or some other relevant resource was shared in place of the existing values. This administrative information
+could also be shared on the Digital Collections website or elsewhere rather than the record. As a convincing argument
+was not made that this information is essential, it was decided to drop these values
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordOrigin"
+
+Decision
+^^^^^^^^
+
+Drop all values.
+
+Here's an `example record - cDanielCartoon:1177 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon%3A1177/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <recordInfo>
+        <recordOrigin>Created and edited in general conformance to MODS Guidelines (Version 3.5).</recordOrigin>
+    </recordInfo>
+
+recordChangeDate
+----------------
+
+Use Case
+^^^^^^^^
+
+This element is used sparingly in UTK's metadata records. Currently there are five distinct values, all indicating that the
+last change to the record was made in 2015, which simply isn't sharing accurate information.
+
+Justification
+^^^^^^^^^^^^^
+
+Keeping this information is not be useful as it doesn't allow someone viewing the record to see when it was actually
+last updated. Inaccurate information is shared. In addition, in a system like Islandora it's easy enough for an internal
+staff member to view when the metadata datastream has been updated without tracking this in the record. This element can
+be dropped.
+
+Xpath
+^^^^^
+
+"mods:recordInfo" / "mods:recordChangeDate"
```

Comment:
Normally, we express an xpath here.  Are you trying to express:

```
mods:recordInfo/mods:recordChangeDate
```

Reply:
This was resolved in https://github.com/UTKcataloging/mods_to_rdf/pull/27/commits/2bd47b57b297b1805e1b9a7e6f517ce517eda056